Tests/UIKitTests/UIStackViewExtensionsTest.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  UIStackViewExtensionsTest.swift
+//  SwifterSwift
+//
+//  Created by Benjamin Meyer on 2/18/18.
+//  Copyright © 2018 SwifterSwift
+//
+
```

Comment:
Hey @BennX  Just one more thing swiftlint is complaining about this extra line here can you please remove it?

Reply:
I'm confused about this, since every other file has the exact same structure.

Reply:
That's strange, I'll check it out :)

Reply:
@LucianoPAlmeida that was right I oversaw that it's in the test file.